packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -772,7 +772,7 @@ async function ensureProjectSettings(
       settings.defaultFunctionName || envDefault[PluginNames.FUNC]?.[defaultFunctionName];
   }
   if (!settings.version) {
```

Comment:
For the case that settings.version is already set to 1.0.0, should we upgrade it as well?

Reply:
Here's an example: https://github.com/OfficeDev/TeamsFx-Samples/blob/main/NPM-search-connector-M365/.fx/settings.json#L18